### PR TITLE
Changing Test function names from camelCase to snake_case

### PR DESCRIPTION
**Pull request by AkashGutha**
This PR is to change camel case function test names to use _ 
This increases readability.

Currently finished changing tests in test_math.py

### CODE REVIEW

**Comment by AkashGutha** [comment]
Yea. that would be great.

File: tests/test_math.py [hunk 1]
```diff
@@ -73,33 +73,33 @@ def testSigmoid(self):
...
+    def test_mat_mul_2d_int(self):
         t1 = TensorBase(np.array([[1, 2],
                                   [3, 4]]))
         t2 = TensorBase(np.array([[5, 6],
                                   [7, 8]]))
         self.assertTrue(syft.equal(syft.matmul(t1, t2), [[19, 22],
                                                          [43, 50]]))
 
-    def testMatmul2DFloat(self):
+    def test_mat_mul_2d_float(self):
```
Comment (aiorla) [inline tests/test_math.py]: :+1: to the idea. Just a tiny comment, I think the MatmulTests should be `test_matmul_X` and not `test_mat_mul_X`.

Comment (AkashGutha) [reply tests/test_math.py]: yes. I agree 👍

**Comment by baldassarreFe** [comment]
I agree on the style change 👍
I've been following the previous style for consistency but I think we should conform to the generally accepted Python guidelines, it will help people jumping on board.

File: tests/test_integrations_locally.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 class PySonarNotebooks(unittest.TestCase):
 
-    def modelTrainingDemoNotebook(self):
+    def model_Training_DemoNotebook(self):
```
Comment (kevinahuber) [inline tests/test_integrations_locally.py]: why not `model_training_demo_notebook`?

File: tests/test_integrations_locally.py [hunk 3]
```diff
@@ -23,7 +23,7 @@ def modelTrainingDemoNotebook(self):
 
 class PySyftNotebooks(unittest.TestCase):
 
-    def paillierHEExampleNotebook(self):
+    def paillier_HEExampleNotebook(self):
```
Comment (kevinahuber) [inline tests/test_integrations_locally.py]: same, `paillier_he_example_notebook`

**Comment by kevinahuber** [comment]
I _believe_ we can enforce this by having `pep8` run on the test files themselves, I'll play around with that today.

File: tests/test_integrations_locally.py (hunk 2 above)
Comment (AkashGutha) [reply tests/test_integrations_locally.py]: I didn't check. I thought it was a file name. so i left it in caps.

File: tests/test_integrations_locally.py (hunk 3 above)
Comment (AkashGutha) [reply tests/test_integrations_locally.py]: same. I'll check and redo these

File: tests/test_integrations_locally.py [hunk 4]
```diff
@@ -23,7 +23,7 @@ def model_Training_DemoNotebook(self):
 
 class PySyftNotebooks(unittest.TestCase):
 
-    def paillier_HEExampleNotebook(self):
+    def paillier_HE_example_notebook(self):
```
Comment (AkashGutha) [inline tests/test_integrations_locally.py]: I'm leaving HE capitalized since it is an abbreviation. 
share any other thoughts on this

**Comment by iamtrask** [comment]
I assume we'll keep waiting on merging this until all the test files have been changed?

**Comment by AkashGutha** [comment]
yes. I'll change the rest too. THen we can merge everything at once.

File: tests/test_tensor.py [hunk 5]
```diff
@@ -561,6 +561,8 @@ def test_chunk_same_size(self):
 
 
 class inequalityTest(unittest.TestCase):
+    # this is an ovverriden function from unittest.Testcase.
```
Comment (AkashGutha) [inline tests/test_tensor.py]: Suggest a better way of writing this warning.

should we be doing something like 
```WARNING : message```

File: tests/test_tensor.py [hunk 6]
```diff
@@ -360,7 +360,7 @@ def test_baddbmm_(self):
 
 
 class PermuteTests(unittest.TestCase):
-    def test_3d(self):
+    def dest3d(self):
```
Comment (AkashGutha) [inline tests/test_tensor.py]: didn't really understand. why changing name didn't work here.
but reverting to the original name worked. so I'm going to leave it as it is.

**Comment by AkashGutha** [comment]
Almost. Done. we can move this once we have it reviewed and fix any remaining issues

**Pull request merged**